pipes/search/src/components/search-chat.tsx

```diff
@@ -676,9 +676,11 @@ export function SearchChat() {
         },
         {
           signal: abortControllerRef.current.signal,
-          headers: {
-            Authorization: `Bearer ${settings.user?.token}`,
-          },
+          ...(settings.aiProviderType === "screenpipe-cloud" && {
+            headers: {
+              Authorization: `Bearer ${settings.user?.token}`,
+            },
```

Comment:
actually we should remove this header thing sicne we already set in the openai sdk

      const openai = new OpenAI({
        apiKey:
          settings.aiProviderType === "screenpipe-cloud"
            ? settings.user.token
            : settings.openaiApiKey,
        baseURL: settings.aiUrl,
        dangerouslyAllowBrowser: true,
      });

Reply:
Thanks. See 5237b2d4b2fe3374bc029b0305331429e24847d9.